pkg/hintrunner/zero/zerohint_others.go

```diff
@@ -30,30 +41,36 @@ func newMemcpyContinueCopyingHint(continueCopying hinter.ResOperander) hinter.Hi
 				return err
 			}
 
-			//> ids.continue_copying = 1 if n > 0 else 0
-			continueCopyingAddr, err := continueCopying.GetAddress(vm)
+			//> ids.continue_loop/continue_copying = 1 if n > 0 else 0
+			continueTargetAddr, err := continueTarget.GetAddress(vm)
 			if err != nil {
 				return err
 			}
 
-			var continueCopyingMv memory.MemoryValue
+			var continueTargetMv memory.MemoryValue
 			if utils.FeltLt(&utils.FeltZero, newN) {
-				continueCopyingMv = memory.MemoryValueFromFieldElement(&utils.FeltOne)
+				continueTargetMv = memory.MemoryValueFromFieldElement(&utils.FeltOne)
 			} else {
-				continueCopyingMv = memory.MemoryValueFromFieldElement(&utils.FeltZero)
+				continueTargetMv = memory.MemoryValueFromFieldElement(&utils.FeltZero)
 			}
 
-			return vm.Memory.WriteToAddress(&continueCopyingAddr, &continueCopyingMv)
+			return vm.Memory.WriteToAddress(&continueTargetAddr, &continueTargetMv)
 		},
 	}
 }
 
-func createMemcpyContinueCopyingHinter(resolver hintReferenceResolver) (hinter.Hinter, error) {
-	continueCopying, err := resolver.GetResOperander("continue_copying")
+func createMemContinueHinter(resolver hintReferenceResolver, memset bool) (hinter.Hinter, error) {
+	var continueTarget hinter.ResOperander
+	var err error
+	if memset {
+		continueTarget, err = resolver.GetResOperander("continue_loop")
+	} else {
+		continueTarget, err = resolver.GetResOperander("continue_copying")
+	}
```

Comment:
Is it better to pass as a parameter to this function the name of the variable instead of a boolean? You could directly fetch this variable from the resolver and you wouldn't have to do the if statement, wdyt? 🤔

Reply:
Also the `newMemContinueHint` function needs to calculate the name of the hint so some sort of `if` condition is still required!